CustomEntitySearchTest/CustomEntitySearchTest.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
Don't think this one needs its own solution. Recommend you either create a Tests folder in the existing PowerSkills sln, and put this project there, or you can create a new top level PowerSkillTests.sln